azurecompute/src/main/java/org/jclouds/azurecompute/binders/DeploymentParamsToXML.java

```diff
@@ -44,27 +44,17 @@
                  .e("ConfigurationSets");
 
          if (params.os() == OSImage.Type.WINDOWS) {
-            XMLBuilder configBuilder = builder.e("ConfigurationSet"); // Windows
+            XMLBuilder configBuilder = builder.e("ConfigurationSet").attr("i:type", "WindowsProvisioningConfigurationSet"); // Windows
             configBuilder.e("ConfigurationSetType").t("WindowsProvisioningConfiguration").up()
                     .e("ComputerName").t(params.name()).up()
                     .e("AdminPassword").t(params.password()).up()
```

Comment:
Notice it is important that first we have to set the `AdminPassword` and after the `WinRM` section `AdminUsername` has to be set.